beacon-chain/state/fieldtrie/field_trie_helpers.go

```diff
@@ -328,3 +330,31 @@ func handleBalanceMVSlice(mv multi_value_slice.MultiValueSliceComposite[uint64],
 	}
 	return [][32]byte{}, nil
 }
+
+func PayloadProof(ctx context.Context, block *blocks.BeaconBlock) ([][]byte, error) {
+	i := block.Body()
+	blockBody, ok := i.(*blocks.BeaconBlockBody)
```

Comment:
If you put this in the `blocks` package, you should be able to write `block.Body` without casting, I think